ecs-agent/go.mod

```diff
@@ -5,37 +5,64 @@ go 1.19
 require (
 	github.com/aws/aws-sdk-go v1.36.0
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
+	github.com/container-storage-interface/spec v1.8.0
 	github.com/didip/tollbooth v4.0.2+incompatible
 	github.com/docker/docker v24.0.2+incompatible
 	github.com/golang/mock v1.4.1
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.5.0
+	github.com/kubernetes-csi/csi-proxy/client v1.1.2
 	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.26.0
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.8.0
-	golang.org/x/sys v0.6.0
-	golang.org/x/tools v0.6.0
+	github.com/prometheus/client_model v0.3.0
```

Comment:
for all these new imports we'll need to validate that there's no collision between the agent go.mod and this one

Reply:
Sry, I accidentally rebase my commits.
I've updated the go.mod and make sure there is no collision for the direct dependencies.